docs/README.md

```diff
@@ -0,0 +1,14 @@
+# Zipkin API
```

Comment:
I think this README belongs where it was, at the root of the project, sam,e for the other files.

The only thing we publish to gh-pages is the swagger stuff

Reply:
Oh. I think this landed on the pages branch unexpectedly. I just put everything into the public folder.

Reply:
ok maybe after digesting the feedback take a close look at files changed etc. the way I do reviews as you are starting to notice is..  evertyihng is liability until it is proven needed ;) so let's try to get what you've accomplished into the smallest diff with the smallest amount of changes. mention me when ready for another pass, and meanwhile, thanks for the help!